README.md

```diff
@@ -1,13 +1,55 @@
 
-SDL Security is a TLS based security library built to interact with sdl_ios and provide TLS certificate authentication and encryption / decryption.
+### How It Works
+SDL Security is an example security library that automotive OEMs can use to build their own proprietary security library. The security library must be configured by the OEM to work with their proprietary version of SDL Core. Once the security library has been configured, it is used to generate a static library that developers add to their SDL iOS apps.
 
-### When is this useful?
-This library can be used to allow TLS authentication on a specific RPC service, for example the RPC or Video service. 
+Anyone implementing this library should take care to add additional protections as this library is not cryptographically secure out-of-the box.
 
-### How do I use it?
-This library is primarily for automotive OEMs to base their own proprietary library on. The OEM may change the URL to retrieve a certificate, and may want to provide additional protection to this library. Otherwise, it will likely be quite easy for an attacker to take the certificate and defeat the TLS protection.
+### What Can Be Encrypted?
+This library can be used to encrypt [SDL services](https://github.com/smartdevicelink/protocol_spec#5-services) such as the video, audio or RPC services.  
 
-The OEM must also rename this library and classes for reasons seen below.
+## Configuring the Library
+The following customizations must be made by the OEM in order for the library to work with their proprietary version of SDL Core.
 
-To use this library, the developer will pass the name of the class into the SDLProxy method `- (void)addSecurityManager:(Class)securityManagerClass forMakes:(NSArray<NSString *> *)vehicleMakes;`. They then also pass in the name of the Vehicle Makes that library is used for, for example, a Ford library may be used for `["Ford", "Lincoln"]`. This must match what is passed through the `register app interface` RPC response.
+### Certificate URL
+The `CertQAURL` URL in the **SDLPrivateSecurityConstants.m** file should to be updated to point to a database that will return certificate data for a specific SDL `appID`. The certificate data will be stored on disk as a `.pfx` file so it can persist between app sessions. If the certificate has expired, the library will automatically try to download a new certificate.
+
+### Vehicle Makes
+The `availableMakes` property should be updated in the **SDLSecurityManager.m** file to list all supported vehicle types. The `vehicleType` returned by SDL Core's `RegisterAppInterface` response is used to select the security manager associated with that vehicle type. It is important that the vehicle types listed in `availableMakes` match exactly the `vehicleType`s returned by the `RegisterAppInterface` response, otherwise the security manager will not be selected and configured. 
```

Comment:
`availableMakes` should probably be moved to `SDLPrivateSecurityConstants`?

Reply:
I created an issue for this: #29

Reply:
I created an issue for this: #29